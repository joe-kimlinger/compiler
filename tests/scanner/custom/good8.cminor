// keywords
ar: array []
f: function()
while (1 > 2)
if (x < 2)
return void
else
print "x", 5, "five"
