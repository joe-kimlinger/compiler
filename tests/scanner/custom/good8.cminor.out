COMMENT
IDENTIFIER ar
COLON
ARRAY
LSQBRACKET
RSQBRACKET
IDENTIFIER f
COLON
FUNCTION
LPAREN
RPAREN
WHILE
LPAREN
INTEGER_LITERAL 1
GT
INTEGER_LITERAL 2
RPAREN
IF
LPAREN
IDENTIFIER x
LT
INTEGER_LITERAL 2
RPAREN
RETURN
VOID
ELSE
PRINT
STRING_LITERAL x
COMMA
INTEGER_LITERAL 5
COMMA
STRING_LITERAL five
